java/doc/api.rst

```diff
@@ -77,15 +71,15 @@ store.
         public <M> M getMeta() throws TaskExecutionException;
     }
 
-This method blocks the current thread until the requested data is ready and has
-been fetched (if necessary) from a remote machine.
+These 2 methods will block the current thread until the requested data is
+locally available.
 
 ``Ray.wait``
 ~~~~~~~~~~~~
 
-Calling ``Ray.wait`` will block the current thread and wait for specified
-ray calls. It returns when at least ``numReturns`` calls are completed,
-or the ``timeout`` expires. See multi-value support for ``RayList``.
+``Ray.wait`` is used to wait for a list of ``RayObject``'s to be locally available.
+It will block the current thread until ``numReturns`` objects are ready or ``timeout``
```

Comment:
The units of the `timeout` should be clarified both here and in the function signature. Really the argument name should be `timeout_milliseconds` or something like that.

Reply:
The parameter in code was already changed to `timeoutMillisecconds`, I'll update this doc accordingly.